dsas-wps/src/main/java/gov/usgs/cida/dsas/wps/CreateTransectsAndIntersectionsProcess.java

```diff
@@ -161,8 +132,6 @@ protected Process(FeatureCollection<SimpleFeatureType, SimpleFeature> shorelines
 			this.store = store;
 			this.transectLayer = transectLayer;
 			this.intersectionLayer = intersectionLayer;
-
-			this.preparedShorelines = null;
```

Comment:
I actually like doing this in the constructor since everything else isn't declared, but I get it.